lib/pharos/autoload.rb

```diff
@@ -47,6 +47,9 @@ module CoreExt
 
   module SSH
     autoload :Client, 'pharos/ssh/client'
+    autoload :Error, 'pharos/ssh/client'
```

Comment:
Why do we need these?

Reply:
`Pharos::SSH::Client` is not yet loaded when the phases are loading.